scripts/_regen_receipt_merge_tree.py

```
@@ -0,0 +1,82 @@
+
+
+def git_cat_blob(tree_sha, path):
+    return subprocess.check_output(
+        ["git", "-C", str(ROOT), "cat-file", "blob", f"{tree_sha}:{path}"],
+        stderr=subprocess.STDOUT,
+    )
+
+
+# Step 1: compute the merge tree
+merge_tree = subprocess.check_output(
+    ["git", "-C", str(ROOT), "merge-tree", "--write-tree", "origin/main", "ext03-exports-metadata"],
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)